x11-drivers/xf86-video-ast/xf86-video-ast-1.1.5.ebuild

```diff
@@ -6,6 +6,6 @@ EAPI=5
 inherit xorg-2
 
 DESCRIPTION="X.Org driver for ASpeedTech cards"
-KEYWORDS="amd64 x86 ~amd64-fbsd ~x86-fbsd"
+KEYWORDS="amd64 x86 ~amd64-fbsd ~x86-fbsd ~ppc64"
```

Comment:
please use the alphabetical order, and use the ekeyword utility from gentoolkit oackage.
Also please use sane commit message https://www.gentoo.org/glep/glep-0066.html